src/components/Chat/react-native-gifted-chat/src/Avatar.js

```diff
@@ -19,6 +19,7 @@ export default class Avatar extends React.Component {
         avatarStyle={StyleSheet
           .flatten([styles[this.props.position].image, this.props.imageStyle[this.props.position]])}
         user={this.props.currentMessage.user}
+        opponentImage={this.props.opponentInfo.profile_picture_small}
```

Comment:
We should check if there's no small picture in DB, if not use profile_picture instead.